yt_dlp/extractor/fujitv.py

```diff
@@ -16,21 +16,32 @@ class FujiTVFODPlus7IE(InfoExtractor):
     }
 
     _TESTS = [{
-        'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810075',
+        'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810076',
         'info_dict': {
-            'id': '5d40810075',
-            'title': '5d40810075',
+            'id': '5d40810076',
             'ext': 'mp4',
-            'format_id': '4000',
-            'thumbnail': 'http://i.fod.fujitv.co.jp/pc/image/wbtn/wbtn_5d40810075.jpg'
+            'title': '#1318 『まる子、まぼろしの洋館を見る』の巻',
+            'series': 'ちびまる子ちゃん',
+            'series_id': '5d40',
+            'description': 'md5:b3f51dbfdda162ac4f789e0ff4d65750',
+            'thumbnail': 'http://i.fod.fujitv.co.jp/img/program/5d40/episode/5d40810076_a.jpg',
         },
-        'skip': 'Expires after a week'
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
+        series_id = self._match_valid_url(url).group('sid')
+        self._download_webpage(url, video_id)
+        # TODO: extract thumbnail from webpage
+        # then remove trim part using regexp: \/(imf\/.*)\/img
+        # where the thumbnail is a img element with id of 'thumbnail'
```

Comment:
don't keep comments like this here.
Either implement this or just remove if doing it is wasteful.

Or you can keep a single line comment something along the lines of  (not necessary btw)
`# Thumbnail can also be obtained from the webpage` but this should go where u are returning/extracting `thumbnail`.